pkg/util/push/otel.go

```diff
@@ -173,6 +173,11 @@ func promToMimirTimeseries(promTs *prompb.TimeSeries) mimirpb.PreallocTimeseries
 		})
 	}
 
+	histograms := make([]mimirpb.Histogram, 0, len(promTs.Histograms))
+	for _, histogram := range promTs.Histograms {
+		histograms = append(histograms, promToMimirHistogram(histogram))
+	}
```

Comment:
[nit] Would it be better to pass pointers into the slice instead of copying histograms into local variables? They seem to be quite big.

```
	for ix := range promTs.Histograms {
		histograms = append(histograms, promToMimirHistogram(&promTs.Histograms[ix]))
	}
```